style.css

```diff
@@ -0,0 +1,1062 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: 'Cocogoose', Helvetica, sans-serif;
+  position: absolute;
+}
+
+.page-padding {
+  padding: 0 80px;
+}
+
+hr {
+  color: rgb(206, 204, 204);
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+  color: #272a31;
+}
+
+a:active {
+  color: #ec5242;
+}
+
+a:hover {
+  color: #ec5242;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+.d-flex {
+  display: flex;
+  flex-direction: row;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.align-center {
+  align-items: center;
+  text-align: center;
+}
+
+@media (max-width: 768px) {
```

Comment:
- [ ] Here I'll try to be more specific. Talking about the default style could be for mobiles is something like this:

```
/* This elements (outside a media parenthesis) are the default */
.head {
 some-element: 3px;
}

@media (min-width: 768px) {
/* All this is for tablet and desktop size */
}
```

And actually this is what you've:

```
/* This elements (outside a media parenthesis) are the default */
.head {
 some-element: 3px;
}

@media (min-width: 768px) {
/* All this is for tablet and desktop size */
}

@media (max-width: 768px) {
/* All inside here (the mobile style), could be at the top without any @media option */
}
```


And that's it, I see on your style you're using the default, 768 px or less and 768 or more, talking about 768px or less is not necessary for a mobile-first design because this could be at the top of the script. Please, make the required changes to have your project according to a mobile-first. 🌵

Reply:
Did not address this!

Reply:
- As the  previous reviewer mentioned your application is not responsive yet I recommend that you apply a mobile-first approach and when your app will be responsive to the mobile size it will be easier to add media queries of the desktop size
This is how your app looks on mobile
![mob](https://user-images.githubusercontent.com/44745855/132848779-488a3b53-9f54-4cec-ae3f-24490b309a7f.png)